contentcuration/contentcuration/assets/channelList/views/Channel/ChannelModal.vue

```diff
@@ -0,0 +1,617 @@
+<template>
```

Comment:
Part of that is that the look inside is the most unstyled part, but also because a lot of the styling from vuetify does a good enough job out of the box.

Reply:
Yeah - as most of that is driven by the `channelDetails` this would make sense, and jives with @jayoshih's suggestion below. And would also stop this being quite so monstrously huge!